documentation/asciidoc/computers/raspberry-pi/boot-msd.adoc

```diff
@@ -2,7 +2,7 @@
 
 NOTE: Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, 4B, 400, Raspberry Pi Zero 2 W, Compute Module 3, Compute Module 3+ and Compute Module 4 only.
```

Comment:
Is it worth changing?
`Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, 4B, 400 and Zero 2 W, and Compute Module 3, 3+ and 4`